portfolio/src/main/webapp/script.js

```diff
@@ -26,8 +27,19 @@ function getQuote() {
 function getMessages() {
     console.log("getting messages");
 
-    fetch('/data').then(response => response.text()).then((messages) => {
-        console.log(messages);
-        document.getElementById('comments-display').insertAdjacentText("afterend", messages);    
+    fetch('/data').then(response => response.json()).then((messages) => {
+        const commentListElement = document.getElementById('comment-list');
```

Comment:
Not sure if this is part of the walkthroughs, but `const` for me conveys that the variable should not be changed, which then contradicts with the appendChild call below.  I would suggest using `let` or `var` instead.